packages/core/component-with-state.ts

```diff
@@ -0,0 +1,63 @@
+import { props } from 'ramda'
+
+import { Component } from './component'
+
+/** @mark draft of types */
+export type StoreDataType = { [key: string]: any }
+export type SelectorType = Array<string>
```

Comment:
selector is function that receives store data and returns sub data - for example `(state) => state.user`

Reply:
Let's reach the consistency with the [data type](https://github.com/one-more/micro-frontends/pull/16#discussion_r329362782) and then I can make it like a function